src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -792,17 +792,46 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
-    <comment>0 - The name of a package reference.</comment>
-  </data>
-  <data name="Warning_CentralPackageVersions_MultipleSourcesWithoutPackageSourceMapping" xml:space="preserve">
+  <data name="Error_CentralPackageManagement_PackageReferenceWithVersionNotAllowed" xml:space="preserve">
+    <value>The following PackageReference items cannot define a value for Version: {0}. Projects using Central Package Management must define a Version value on a PackageVersion item. For more information, visit https://aka.ms/nuget/cpm/gettingstarted</value>
+    <comment>
+      0 - Comma delimited list of the package IDs that specified a Version
+      Do not localize `PackageReference`
+      Do not localize `PackageVersion`
+    </comment>
+  </data>
+  <data name="Error_CentralPackageManagement_ImplicitPackageReferenceWithVersionNotAllowed" xml:space="preserve">
+    <value>The following PackageReference items are implicitly defined and cannot define a PackageVersion item: {0}. Projects using Central Package Management require that implicit package versions be specified by the PackageReference item. For more information, visit https://aka.ms/sdkimplicitrefs</value>
+    <comment>
+      0 - Comma delimited list of the package IDs that were implicitly defined and should not have a PackageVersion item
+      Do not localize `PackageReference`
+      Do not localize `PackageVersion`
+  </comment>
+  </data>
+  <data name="Error_CentralPackageManagement_MissingPackageVersion" xml:space="preserve">
+    <value>The following PackageReference items do not define a corresponding PackageVersion item: {0}. Projects using Central Package Management must declare PackageReference and PackageVersion items with matching names. For more information, visit https://aka.ms/nuget/cpm/gettingstarted</value>
+    <comment>
+      0 - The comma delimited list of package names that did not have a version specified.
+      Do not localize `PackageReference`
+      Do not localize `PackageVersion`
+    </comment>
+  </data>
+  <data name="Error_CentralPackageManagement_FloatingVersionsNotAllowed" xml:space="preserve">
+    <value>The following PackageVersion items cannot specify a floating version: {0}. For more information on how to enable this functionality for projects using Central Package Management, visit https://aka.ms/nu1011</value>
+    <comment>
+      0 - The comma delimited list of package names that specified a floating version.
+      Do not localize `PackageVersion`
+  </comment>
+  </data>
+  <data name="Error_CentralPackageManagement_VersionOverrideNotAllowed" xml:space="preserve">
+    <value>The following PackageReference items cannot specify a value for VersionOverride: {0}.  Projects using Central Package Management are currently configured to disable this functionality. For more information, visit https://aka.ms/nuget/cpm/versionoverride</value>
+    <comment>
+      0 - The comma delimited list of package names that specified a value for VersionOverride.
```

Comment:
Just a reminder that we currently have no way of propagating comments to the localization team. Might need to reach out to them or file a ticket to ensure these aren't localized.